scripts/dev_setup.sh

```diff
@@ -15,7 +15,7 @@ SHELLCHECK_VERSION=0.7.1
 HADOLINT_VERSION=1.17.4
 SCCACHE_VERSION=0.2.14-alpha.0
 #If installing sccache from a git repp set url@revision.
-SCCACHE_GIT='https://github.com/rexhoffman/sccache.git@19fef99c15765ea73460fd9ecb209c35313eac41'
+SCCACHE_GIT='https://github.com/rexhoffman/sccache.git@549babdd3866aa60dae01668c42ee00bf1e8c763'
```

Comment:
Should have done this yesterday when I made the pr on sccache mergeable again -- and dev updated more deps today with more collisions.... 🤷